### DISCUSSION TITLE
indent  - shift + alt + f - things that don't seem right to me.

### DISCUSSION THREAD

**Discussion by WZDondoni** [prettier/prettier#15390] [Q&A] [answered]
First tks for the extension. 
Second. Hello, I'm learnig.
When i use the shortcut, the code organizes itself, but with things that don't seem right to me.
![image](https://github.com/prettier/prettier/assets/81537208/82ef8630-4f86-49e5-b3da-fc90c5dd54c1)

**Comment by so1ve** [comment, marked as answer]
This is intended; checkout the `htmlWhitespaceSensitivity` option.
